opentech/templates/includes/message_item.html

```diff
@@ -1,8 +1,9 @@
 <li class="messages__text{% if tag %} messages__text--{{ tag }}{% endif %}{% if close %} js-message{% endif %}">
-    {{ message }}
-    {% if close %}
-    <a href="#" class="messages__close js-close-message">
-        <svg class="icon icon--close"><use xlink:href="#cross"></use></svg>
-    </a>
-    {% endif %}
+    <div class="messages__inner">
+        <svg class="messages__icon"><use xlink:href="#exclamation-point"></use></svg>
```

Comment:
this svg can have aria-hidden="true" as it isn't necessary for understanding the text. According to the css tricks accessible svg articles it should also have a title tag, despite being aria-hidden.